### PR TITLE
#20367 extend error message if initialize client ini file ends with error

### PR DESCRIPTION
**Pull request by shecken**

### CODE REVIEW

File: Services/Init/classes/class.ilInitialisation.php [hunk 1]
```diff
@@ -311,7 +311,7 @@ protected static function initClientIniFile()
 			}
 			else
 			{
-				self::abortAndDie("Invalid client");
+				self::abortAndDie("Fatal Error: ilInitialisation::initClientIniFile initializing client ini file abborted with: ". $ilClientIniFile->ERROR);
```
Comment (klees) [inline Services/Init/classes/class.ilInitialisation.php:314]: Did you check that there is anything meaningful in `$ilClientIniFile->ERROR`?

Comment (shecken) [reply Services/Init/classes/class.ilInitialisation.php:314]: Yes i did. E.g if the file does not exist ne message will be like this.

`Fatal Error: ilInitialisation::initClientIniFile initializing client ini file abborted with: file_does_not_exist`

**Review by klees — APPROVED** [review]
LGTM

**Comment by alex40724** [comment]
Thanks! Merged to 5.0,5.1,5.2

**Pull request merged**